invenio_app_ils/documents/schemas/documents/document-v1.0.0.json

```diff
@@ -706,7 +706,7 @@
       "uniqueItems": true
     }
   },
-  "required": ["$schema", "pid", "title", "authors", "publication_year"],
+  "required": ["$schema", "pid", "title", "authors", "publication_year", "document_type"],
```

Comment:
Added `document_type` as required, ping @kprzerwa

Reply:
it would be nice to have a default as `BOOK` maybe?

Reply:
I didn't want because there enums are meant to be overridable if you extend the class. If we hardcoded in the schema, then it is maybe harder to change? WDYT?

Reply:
ok for me, I can just add it in the migration